File: myCTCA Mobile/Android/src/myctca/app/src/main/java/com/myctca/model/PrescriptionRefillRequest.java
package com.myctca.model;

import java.util.ArrayList;
import java.util.List;

public class PrescriptionRefillRequest {

    public List<String> selectedPrescriptions = new ArrayList<>();
    public String patientPhone;
    public String pharmacyName;
    public String pharmacyPhone;
    public String comments;
    public List<CareTeam> to;

}

File: myCTCA Mobile/iOS/src/myctca/Models/PrescriptionRefillRequest.swift
//
//  PrescriptionRefillRequest.swift
//  myctca
//
//  Created by Tomack, Barry on 3/29/18.
//  Copyright © 2018 CTCA. All rights reserved.
//

import UIKit

struct PrescriptionRefillRequest : Encodable {

    var selectedPrescriptions: [String] = [String]()
    var patientPhone: String?
    var pharmacyName: String?
    var pharmacyPhone: String?
    var comments: String?
    var to:[CareTeam] = []
    
    init() { }
}

